hnn_core/network.py

```diff
@@ -484,6 +466,35 @@ def plot(self, ax=None, show=True):
         """
         return plot_spikes_raster(spikes=self, ax=ax, show=show)
 
+    def plot_hist(self, ax=None, spike_types=None, show=True):
+        """Plot the histogram of spiking activity.
+
+        Parameters
+        ----------
+        ax : instance of matplotlib axis | None
+            An axis object from matplotlib. If None,
+            a new figure is created.
+        spike_types: string | list | dictionary | None
+            String input of a valid spike type is plotted individually.
+                Ex: 'common', 'evdist', 'evprox', 'extgauss', 'extpois'
+            List of valid string inputs will plot each spike type individually.
+                Ex: ['common', 'evdist']
+            Dictionary of valid lists will plot list elements as a group.
+                Ex: {'Evoked': ['evdist', 'evprox'], 'External': ['extpois']}
+            If None, all input spike types are plotted individually.
+            Valid strings also include leading characters of spike types
+                Example: 'ext' is equivalent to ['extgauss', 'extpois']
+        show : bool
+            If True, show the figure.
+
+        Returns
+        -------
+        fig : instance of matplotlib Figure
+            The matplotlib figure handle.
+        """
+        return plot_hist(
```

Comment:
I would use a more descriptive name in viz.py. Something like plot_spikes_hist ?

so it's a bit clear that it plots histograms only for spikes objects